stdlib/2.7/socket.pyi

```diff
@@ -278,7 +278,7 @@ class socket:
     proto = 0
 
     def __init__(self, family: int = ..., type: int = ...,
-                 proto: int = ..., fileno: int = ...) -> None: ...
+                 proto: int = ..., fileno: Optional[int] = ...) -> None: ...
```

Comment:
Yeah, the _sock argument is not an int, unlike the PY3 file no argument.

--Guido (mobile)